examples/lang/exec_duplicate.mcl

```diff
@@ -0,0 +1,7 @@
+exec "exec_duplicate" {
+	cmd => "/usr/bin/env",
```

Comment:
Why do we want to add an example of what's not allowed?

Reply:
i could have sworn you told me to, but if you don't want it I'll just remove it.